LEAF_Request_Portal/js/combined_inbox_editor/src/combined_inbox_editor.vue.js

```diff
@@ -153,24 +153,37 @@ const CombinedInboxEditor = Vue.createApp({
                 resolve();
             });
         },
-        async getIndicators(site) {
-            if (typeof site == 'undefined' || typeof site.target == 'undefined') {
+        async getIndicators(portalURL = '') {
+            if (portalURL === '') {
                 return [];
             }
-            const indicators = await fetch(site.target + "api/form/indicator/list", {
+            const indicators = await fetch(portalURL + "api/form/indicator/list", {
                 headers: {
                     "Content-Type": "application/json",
                 },
                 cache: "no-cache"
             });
-    
-            return indicators.json();
+            return await indicators.json();
         }
     },
     created() {
         this.getMapSites().then(() => {
             this.sites.forEach((site) => {
-                this.getIndicators(site).then((indicators) => {
+                let portalURL = site.target;
+                if(portalURL.indexOf('/admin/') != -1) {

```

Comment:
Is this going to run into the same issues Jamie has been for this "admin" name?

Reply:
It shouldn't because of the "/" on either side.

Reply:
https://localhost/LEAF_Request_Portal/admin/?a=formLibrary not 100% sure what is going on with the admin stuff right now, however that is a url I just grabbed off of local with the slashes both sides.

Reply:
The above example would be fine as a card for what this ticket is addressing, because at this point it's trying to get the site's base URL to query for requests to display in the inbox.  The local formLibrary view has its own issues not related to this one (we don't have a local LEAF/library portal).

However, there is one portal named 'admin'.  They currently have no sitemap cards, but the inbox and other pages that make queries for requests based on card URLs would error if they were to add one for their portal or, -- more problematically -- if they were linked to with cards by other sites.  Since these URLs are from sitemap cards, it's not certain that the nexus of the site parsing the card and the nexus part of the card are the same (otherwise we could look for /admin/ not immediately preceded by the nexus).

Reply:
closing per AM discussion.